hasher-matcher-actioner/hmalib/scripts/migrations/2022_04_02_default_content_signal_type_configs.py

```diff
@@ -13,40 +14,62 @@
     ToggleableSignalTypeConfig,
     full_class_name,
 )
-
+from hmalib.common.logging import get_logger
 from hmalib.scripts.migrations.migrations_base import MigrationBase
 
+logger = get_logger(__name__)
+
 
 class _Migration(MigrationBase):
```

Comment:
Should we take a step to rename all these migration things to try and separate "Setup" from "Migration for compatibility?"

I'm especially worried about folks getting state re-applied to them after they explicitly opt out. 

If this is already handled in some way, can we separate these two in the codebase by calling one InitialSetup and another MigrateOldDeployments or somesuch?

Reply:
It is handled another way, if they use terraform it'll be added once but can be removed. If removed it will not be applied because the terraform data won't have changed.

Reply:
As for the rename, we probably want to take the step eventually however I didn't want to make that drastic a rewrite without a lot more testing and I wanted to un-break the regular deploy first anyway. I am happy to create or have you create and issue to follow through with a rewrite.